docs/release-notes-pxc-8.4.0.md

```diff
@@ -0,0 +1,27 @@
+# Percona Distribution for MySQL 8.4.0 using Percona XtraDB Cluster (2024-11-12)
+
+Percona Distribution for MySQL is the most stable, scalable, and secure open source MySQL distribution, with two download options: one based on Percona Server for MySQL and one based on Percona XtraDB Cluster. [Install Percona Distribution for MySQL](installing.md).
+
+This release is based on [Percona XtraDB Cluster 8.4.0-1](https://docs.percona.com/percona-xtradb-cluster/8.4/release-notes/8.4.0-1.html).
+
+## Release highlights
+
+Percona XtraDB Cluster is based on Percona Server for MySQL. Find a complete list of improvements and bug fixes in the [Percona Server for MySQL 8.4.0-1 (2024-08-28) release notes](https://docs.percona.com/percona-server/8.4/release-notes/8.4.0-1.html).
+
+## Supplied components
+
+Review each component’s release notes for What’s new, improvements, or bug fixes. The following is a list of the components supplied with the Percona XtraDB Cluster-based variation of the Percona Distribution for MySQL:
+
+!!! important
+
+    This release does not include the Percona Toolkit component. We will add it once a Percona Toolkit version compatible with MySQL 8.4 becomes available.
```

Comment:
We could move these paragraphs to Known issues to highlight them better.

Reply:
Good idea. Thank you!